kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/Client.java

```diff
@@ -189,4 +189,23 @@ default <T extends HasMetadata, L extends KubernetesResourceList<T>> MixedOperat
 
   Config getConfiguration();
 
+  /**
+   * GET the response from the given uri as a String
+   *
+   * @param uri must start with / if relative
+   * @return the response, or null if a 404 code
+   */
+  default String raw(String uri) {
```

Comment:
If it didn't entail a security issue I'd suggest to use `URI` as the parameter type instead.